RELEASE_NOTES.rst

```diff
@@ -217,13 +216,14 @@ Bugfixes:
    ``stderr``.
    (https://github.com/coala-analyzer/coala-bears/issues/251)
 
+
 coala-bears 0.1.0 beta
 =======================
 
 coala-bears is a Python package containing all the bears that are used by coala.
 It has been split from `coala <https://github.com/coala-analyzer/coala>`_.
 With the initial release, it features 56 bears covering 32 languages.
-You can see all of them `here <https://gist.github.com/Adrianzatreanu/cf2d0c8b2ecd542a4860>`_
```

Comment:
I believe that rstcheck is being very pedantic if it is warning about this.

However "here" is overused and disliked as a link text.  But "changes here" is not a huge improvement.

"on this gist" might be better, as it gives the reader a little more information about what the link is.